package.json

```diff
@@ -2,14 +2,14 @@
   "name": "eslint-config-google",
   "version": "0.6.0",
   "description": "ESLint shareable config for the Google style",
-  "license": "Apache-2",
+  "license": "Apache-2.0",
   "repository": "google/eslint-config-google",
   "author": "Google",
   "engines": {
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "ava"
+    "test": "node test/test.js"
```

Comment:
Thanks for the insight. I consider it higher-pri for us to land support for the new styleguide so let's move ahead with the change in the testing setup. If once we move to more complex tests we find that Ava would be worth bringing back in, it would be great to consider it.